hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -512,6 +513,7 @@ long acquireLock() throws UnknownHostException, TException, InterruptedException
     LockResponse lockResponse = metaClients.run(client -> client.lock(lockRequest));
     AtomicReference<LockState> state = new AtomicReference<>(lockResponse.getState());
     long lockId = lockResponse.getLockid();
+    final Pair<Long, String> lockDetails = Pair.of(lockId, String.format("%s.%s", database, tableName));
```

Comment:
I do not think we need this `Pair` object here. Could we just use database/tableName down below?

Reply:
I think the `Tasks.foreach(lockDetails)` is only used to reuse the retry functionality of `Tasks`. So I think it is perfectly fine to keep the simpler implementation and using database/tableName.